api/proto/teleport/integration/v1/awsoidc_service.proto

```diff
@@ -236,6 +239,53 @@ message DeployDatabaseServiceResponse {
   string cluster_dashboard_url = 2;
 }
 
+// DeployServiceRequest is a request to deploy .
+message DeployServiceRequest {
+  // Integration is the AWS OIDC Integration name.
+  // Required.
+  string integration = 1;
+  // Region is the AWS Region
+  // Required.
+  string region = 2;
+  // DeploymentMode is the deployment name that should be applied when creating the ECS Service.
+  // Allowed modes: database-service
+  // Required.
+  string deployment_mode = 3;
+  // SecurityGroups to apply to the service's network configuration.
+  // If empty, the default security group for the VPC is going to be used.
+  repeated string security_groups = 4;
+  // SubnetIds are the subnets for the network configuration.
+  // Required.
+  repeated string subnet_ids = 5;
+  // TaskRoleARN is the AWS IAM Role received by the deployed service.
+  // Required.
+  string task_role_arn = 6;
+  // TeleportVersion is the teleport version to be deployed.
+  // This is used to fetch the correct tag for the teleport container image.
+  // Eg, 14.3.4 (no "v" prefix)
+  // Required.
+  string teleport_version = 7;
+  // DeploymentJoinTokenName is the Teleport IAM Join Token to be used by the deployed
+  // service to join the cluster.
+  // Required.
+  string deployment_join_token_name = 8;
+  // DeployServiceConfig is the teleport.yaml configuration (base64 encoded) used by teleport.
+  // Required.
+  string deploy_service_config = 9;
```

Comment:
Nit: `teleport_config_string` would be consistent with `DeployDatabaseServiceDeployment` and probably a more descriptive name for this.

Reply:
Yeah, looks better.
Changed